indicoio/api/prebuilt.py

```diff
@@ -4,13 +4,35 @@
 
 from indicoio.preprocess.pdf import pdf_preprocess
 from indicoio.errors import IndicoInputError
+from indicoio.client.storage import StorageClient
 from typing import List
 
 
 def _convert_options_to_str(options):
     return ",".join(f"{key}: {json.dumps(option)}" for key, option in options.items())
 
 
+def enforce_list(inputs):
+    if not isinstance(inputs, list):
+        raise IndicoInputError(
+            "This function expects a list input. If you have a single piece of data, please wrap it in a list"
+        )
+
+
+def _convert_files_meta_str(uploaded_files: List[dict]):
+    file_inputs = [
+        {"name": f["name"], "path": f["path"], "uploadType": f["type"]}
+        for f in uploaded_files
+    ]
+
+    str_files = [
```

Comment:
why aren't we just doing json.dumps of file_inputs

Reply:
For it to be a valid argument for the graphql mutation, arguments need to not be quoted: 
`{arg1: "value1", arg2: "value2"}`

json.dumps produces
`{"arg1": "value1", "arg2", "value2"}`

Reply:
That should not be true.

Reply:
This is an example of the mutation being constructed. 
```mutation {
  documentExtraction(files: [{path: "fsdf", name: "sdsf", uploadType: "dfgds"}], data:"dgs")
  {
    jobId
  }

}
```
If I put quotes around the `path` for example, it will be a graphql synax error.

Reply:
Your fog is incorrect then. Also, there' already a mutation for this in the workflows branch.

Reply:
FileInput should be:
filename=String()
filemeta=JSONString()

filemeta should be [{"path": "fsdf", "name": "sdsf", "uploadType": "dfgds"}]
filename should be: name of file

Reply:
I'm pretty sure if I pass them in as variable, variables can be a json. Will try this.

Reply:
After updating the fog schema with the JSONString as one of the fields in the FileInput object, there is still parsing required on top of `json.dumps`. For the JSONString portion, that can be sent as input cleanly. However, `filename` and `filemeta` need quotes removed since they are not part of the json object.